nntrainer/models/neuralnet.cpp

```diff
@@ -70,7 +70,7 @@ NeuralNetwork::NeuralNetwork() :
     props::Epochs(), props::TrainingBatchSize(), props::SavePath(),
     props::ContinueTrain(), props::SaveBestPath(), props::MemoryOptimization(),
     props::MemorySwap(), props::MemorySwapPath(), props::MemorySwapLookahead(),
-    props::TensorFormat(), props::ModelTensorDataType()),
+    props::TensorFormat(), props::ModelTensorDataType(), props::LossScale()),
```

Comment:
I wonder whether props::LossScale is layer common property or network comment property. If it is network common property, then it should be set in ```model/model_common_properties.h```.

Reply:
It used for both, so it may be better with just common property.
If a user set it as model property, train core will set the value to layer property dynamically.

Reply:
> It used for both, so it may be better with just common property.
> If a user set it as model property, train core will set the value to layer property dynamically.

=> It moved to model property.